study-builder/studies/angio/emails/blood_received.html

```diff
@@ -0,0 +1,49 @@
+<html>
+<head>
+  <title>Thank you, your blood kit was received</title>
```

Comment:
These emails are copied from what we have in SendGrid. Checking them into git so we have visibility into them. `<title>` is the subject line of the email.